.jenkins/Jenkinsfile

```diff
@@ -351,38 +355,50 @@ properties([buildDiscarder(logRotator(artifactDaysToKeepStr: '90',
-         "ACC1804 clang-7 Debug" :                              { ACCTest('ACC-1804', 'clang-7', 'Debug') },
-         "ACC1804 clang-7 Release" :                            { ACCTest('ACC-1804', 'clang-7', 'Release') },
-         "ACC1804 gcc Debug" :                                  { ACCTest('ACC-1804', 'gcc', 'Debug') },
-         "ACC1804 gcc Release" :                                { ACCTest('ACC-1804', 'gcc', 'Release') },
-         "ACC1804 Container RelWithDebInfo" :                   { ACCContainerTest('ACC-1804', '18.04') },
-         "ACC1804 Package RelWithDebInfo" :                     { ACCPackageTest('ACC-1804', '18.04') },
-         "ACC1804 GNU gcc SGX1FLC" :                            { ACCGNUTest() },
-         "AArch64 1604 GNU gcc Debug" :                         { AArch64GNUTest('16.04', 'Debug')},
-         "AArch64 1604 GNU gcc Release" :                       { AArch64GNUTest('16.04', 'Release')},
-         "AArch64 1804 GNU gcc Debug" :                         { AArch64GNUTest('18.04', 'Debug')},
-         "AArch64 1804 GNU gcc Release" :                       { AArch64GNUTest('18.04', 'Release')},
-         "Sim 1804 clang-7 SGX1 Debug" :                        { simulationTest('18.04', 'SGX1', 'Debug')},
-         "Sim 1804 clang-7 SGX1 Release" :                      { simulationTest('18.04', 'SGX1', 'Release')},
-         "Sim 1804 clang-7 SGX1-FLC Debug" :                    { simulationTest('18.04', 'SGX1FLC', 'Debug')},
-         "Sim 1804 clang-7 SGX1-FLC Release" :                  { simulationTest('18.04', 'SGX1FLC', 'Release')},
-         "Win2016 Ubuntu1604 clang-7 Debug Linux-Elf-build" :   { win2016LinuxElfBuild('16.04', 'clang-7', 'Debug') },
-         "Win2016 Ubuntu1604 clang-7 Release Linux-Elf-build" : { win2016LinuxElfBuild('16.04', 'clang-7', 'Release') },
-         "Win2016 Ubuntu1804 clang-7 Debug Linux-Elf-build" :   { win2016LinuxElfBuild('18.04', 'clang-7', 'Debug') },
-         "Win2016 Ubuntu1804 clang-7 Release Linux-Elf-build" : { win2016LinuxElfBuild('18.04', 'clang-7', 'Release') },
-         "Win2016 Ubuntu1804 gcc Debug Linux-Elf-build" :       { win2016LinuxElfBuild('18.04', 'gcc', 'Debug') },
-         "Win2016 Debug Cross Compile" :                        { win2016CrossCompile('Debug') },
-         "Win2016 Release Cross Compile" :                      { win2016CrossCompile('Release') },
-         "Win2016 Debug Cross Compile with DCAP libs" :         { win2016CrossCompile('Debug', 'ON') },
-         "Win2016 Release Cross Compile with DCAP libs" :       { win2016CrossCompile('Release', 'ON') },
-         "Host verification Debug" :                            { ACCHostVerificationTest('18.04', 'Debug') },
-         "Host verification Release" :                          { ACCHostVerificationTest('18.04', 'Release') }
+try{
+    oe.emailJobStatus('STARTED')
+
+    parallel "Check Developer Experience Ubuntu 16.04" :            { checkDevFlows('16.04') },
```

Comment:
Nit: fix whitespace alignment

For my understanding, does a failure in a parallel job trigger a single exception that is mailed or a swath of all subsequent jobs being aborted create a race condition on the exception that actually gets captured by `GLOBAL_ERROR`?

Reply:
Hey @CodeMonkeyLeet,

Checkout https://github.com/openenclave/openenclave/pull/2310#discussion_r345950041

I had the same questions and the job testing this scenario was posted.

Reply:
The best example illustrating this behavior is from the below job sequence - which contains multiple stages run in parallel:

![image](https://user-images.githubusercontent.com/56414747/68798897-3fabc600-0625-11ea-9bc2-6b862d2f5a32.png)

Multiple stages fail (by subscript execution in oe-ci library), while multiple stages succeed.  The email is not sent per stage failure, but only at the end.

![image](https://user-images.githubusercontent.com/56414747/68799024-797ccc80-0625-11ea-8d2f-60fb80243113.png)

![image](https://user-images.githubusercontent.com/56414747/68799045-800b4400-0625-11ea-83b1-41d7c8e4d073.png)

![image](https://user-images.githubusercontent.com/56414747/68799068-88fc1580-0625-11ea-8c02-9a325368ac11.png)